### PR TITLE
DataTokenizationExample_avro

### PR DESCRIPTION
**Pull request by daria-malkova**
AVRO support in data tokenization example

### CODE REVIEW

File: examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java [hunk 1]
```diff
@@ -260,8 +261,7 @@ public static PipelineResult run(DataTokenizationOptions options) {
         rows = rows.apply(Window.into(FixedWindows.of(parseDuration(options.getWindowDuration()))));
       }
     } else {
-      throw new IllegalStateException(
-          "No source is provided, please configure File System or Pub/Sub");
+      throw new IllegalStateException("No source is provided, please configure GCS or Pub/Sub");
```
Comment (KhaninArtur) [inline examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java:264]: Why is it only GCS? Can't we support any FS in Beam?

File: examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/transforms/io/TokenizationFileSystemIO.java [hunk 2]
```diff
@@ -141,12 +149,26 @@ public TokenizationFileSystemIO(DataTokenizationOptions options) {
...
+    org.apache.avro.Schema avroSchema = AvroUtils.toAvroSchema(schema.getBeamSchema());
+    PCollection<GenericRecord> genericRecords = pipeline.apply(
+        "ReadAvroFiles",
+        AvroIO.readGenericRecords(avroSchema).from(options.getInputFilePattern()));
+    return genericRecords
+        .apply(
+            "GenericRecordToRow", MapElements.into(TypeDescriptor.of(Row.class))
+                .via(AvroUtils.getGenericRecordToRowFunction(schema.getBeamSchema())))
+        .setCoder(RowCoder.of(schema.getBeamSchema()));
+  }
```
Comment (KhaninArtur) [inline examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/transforms/io/TokenizationFileSystemIO.java:170]: Let's merge these 2 parts into 1 the same way it is done in `writeAvro` function.

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java [hunk 3]
```diff
@@ -22,6 +22,7 @@
 import static org.apache.beam.vendor.guava.v26_0_jre.com.google.common.base.Preconditions.checkArgument;
 
 import java.io.IOException;
+import java.io.Serializable;
```
Comment (ilya-kozyrev) [inline examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java:25]: remove unnecessary imports

**Review by ilya-kozyrev — CHANGES_REQUESTED** [review]

File: examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java (hunk 3 above)
Comment (daria-malkova) [reply examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java]: done

File: examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/transforms/io/TokenizationFileSystemIO.java (hunk 2 above)
Comment (daria-malkova) [reply examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/transforms/io/TokenizationFileSystemIO.java]: done

File: examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java (hunk 1 above)
Comment (daria-malkova) [reply examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/DataTokenization.java]: done

**Review by KhaninArtur — APPROVED** [review]
LGTM, thanks!

**Pull request closed without merging**